tutorials/application_design/py/README.md

```diff
@@ -0,0 +1,47 @@
+# Tutorial: Data Persistence
+
+This code is part of the Connext Application Design tutorial and is included
+here in full for convenience.
+Please see the tutorial for instructions.
+
+This code targets Python 3.8 and uses the Connext Python API.
+
+## How to build
+
+The example already provides generated types (`VehicleModeling.py`), so nothing
+needs to be done for Python applications.
+
+## How to run
+
+The publisher can be run as a file or module. The expected output contains a
```

Comment:
For consistency with the rest of this repo, provide the instructions to run as a file only.